types/historykana/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for HistoryKana 1.0
+// Project: https://github.com/terrierscript/historykana
+// Definitions by: shiratsuki <https://github.com/h-shiratsuki>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+/**
+ * Get Kana string from input history array.
+ * @param histories Input History Array
+ * @param options kanaRegexp : Hiragana detection regular expression rule.
+ */
+export default function historykana(
```

Comment:
This looks like this should be [export equals](https://github.com/terrierscript/historykana/blob/master/index.js)

Reply:
"No-single-declare-module" error and "no-declare-current-package" error occurred when changing to "export =".
In order to solve the above problem, I changed it to "export function" and changed the import to "import {x}", but it can not be read module at runtime and will result in an error.

Reply:
Try along the lines of
```ts
declare function historykana(...stuff...): ...stuff...
export = historykana;
```

Reply:
Thanks! This worked without problems.